src/components/Aside/Aside.js

```diff
@@ -7,31 +7,53 @@ import {
   MdHistory,
 } from "react-icons/md";
 import { IoMdHeart } from "react-icons/io";
+import { NavLink } from "react-router-dom";
 
 const Aside = () => {
   return (
     <>
       <aside className="aside">
         <ul>
-          <li className="active">
+          
+          <li>
+            <NavLink to='/'  className={({ isActive }) =>
```

Comment:
instead of giving every NavLink separate style you can make a one function for this,  and pass into style attribute

Reply:
Okie will do it.